src/hyp3_isce2/packaging.py

```diff
@@ -402,18 +401,23 @@ def make_parameter_file(
         sec_tag = secondary_scene[-4::]
 
     if reference_safe is None:
-        reference_safe = [file for file in os.listdir('.') if file.endswith(f'{ref_tag}.SAFE')][0]
-        secondary_safe = [file for file in os.listdir('.') if file.endswith(f'{sec_tag}.SAFE')][0]
-
-    assert reference_safe and secondary_safe
-
-    ref_annotation_path = f'{reference_safe}/annotation/'
-    ref_annotation = [file for file in os.listdir(ref_annotation_path) if os.path.isfile(ref_annotation_path + file)][0]
+        reference_safe_list = list(Path.cwd().glob(f'*{ref_tag}.SAFE'))
+        assert len(reference_safe_list) == 1, f'Expected one reference SAFE, found {len(reference_safe_list)}'
+        reference_safe = reference_safe[0]
```

Comment:
First item of the list, yes? Not first character of the safe?
```suggestion
        reference_safe = reference_safe_list[0]
```